packages/dds/tree/src/test/shared-tree/sharedTree.bench.ts

```diff
@@ -431,4 +432,69 @@ describe("SharedTree benchmarks", () => {
 			}
 		}
 	});
+
+	// In this context "op bunch" refers to a group of ops for the same DDS that are sent by a peer in a single message.
+	describe("rebasing over op bunch", () => {
+		// The number of commits in a bunch for a given run of this test suite.
+		const bunchSizes = isInPerformanceTestingMode ? [1, 10] : [2];
+		// Number of local commits to rebase over the inbound bunch
+		const localBranchSizes = isInPerformanceTestingMode ? [10, 100] : [2];
+		// If op bunching is used, then the time taken to rebase the local branch over the bunch should be proportional to `bunchSize + localBranchSize` instead of `bunchSize * localBranchSize`.
+		// For the following timings:
+		// +----------------------+-------------+--------------+
+		// |                      | bunchSize:1 | bunchSize:10 |
+		// +----------------------+-------------+--------------+
+		// | localBranchSize: 10  | t1          | t2           |
+		// | localBranchSize: 100 | t3          | t4           |
+		// +----------------------+-------------+--------------+
+		// Then, if op bunching is used, then t4 should be roughly equal to t1 + (t2-t1) + (t3-t1)
+		for (const bunchSize of bunchSizes) {
+			for (const localBranchSize of localBranchSizes) {
+				const test = benchmark({
+					type: BenchmarkType.Measurement,
+					title: `Rebase ${localBranchSize} local commits over ${bunchSize} inbound commits`,
+					benchmarkFnCustom: async <T>(state: BenchmarkTimer<T>) => {
+						let duration: number;
+						do {
+							// Since this setup one collects data from one iteration, assert that this is what is expected.
+							assert.equal(state.iterationsPerBatch, 1);
+							const provider = new TestTreeProviderLite(
+								2,
+								factory,
+								undefined /* useDeterministicSessionIds */,
+								FlushMode.TurnBased,
+							);
+							const sender = provider.trees[0];
+							const receiver = provider.trees[1];
+							// Add commits to the receiver's local branch but prevent them from being sent in order to ensure they remain on the local branch
+							receiver.setConnected(false);
+							for (let iCommit = 0; iCommit < localBranchSize; iCommit++) {
+								insert(receiver.checkout, 0, `r${iCommit}`);
+							}
+							// These are the commits that should be bunched together
+							for (let iCommit = 0; iCommit < bunchSize; iCommit++) {
+								insert(sender.checkout, 0, `s${iCommit}`);
+							}
+							// Ensure the sender has sent the ops
+							provider.processMessages();
+							// Prevent the sender from receiving anything else since we only want to measure the rebase on the receiver
+							sender.setConnected(false);
+							const before = state.timer.now();
+							// Allow the receiver to receive the bunched commits.
+							// This should force the local branch to be rebased over the bunch.
+							receiver.setConnected(true);
+							provider.processMessages();
```

Comment:
Let's remove `provider.processMessages();` since that only adds the processing of the receiver's edits.